apps/ui/package.json

```
@@ -28,14 +28,20 @@
     "@graphql-debugger/time": "workspace:^",
     "@graphql-debugger/utils": "workspace:^",
     "@tanstack/react-query": "5.4.3",
+    "@testing-library/jest-dom": "^6.1.4",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.5.1",
     "classnames": "2.3.2",
+    "jest-environment-jsdom": "^29.7.0",
+    "jest-transformer-svg": "^2.0.1",
```

```suggestion
    "@testing-library/jest-dom": "6.1.4",
    "@testing-library/react": "14.0.0",
    "@testing-library/user-event": "14.5.1",
    "classnames": "2.3.2",
    "jest-environment-jsdom": "29.7.0",
    "jest-transformer-svg": "2.0.1",
```

Can we pin these dependencies, please?  

We should add an eslint rule that enforces this in another PR. 

You might have to delete the pnpm-lock and install again after pining.

---

Dependabot updates them